### PR TITLE
Handle search result and redirect to translate page

### PR DESCRIPTION
**Pull request by samwilson**
This adds the stub of the translate page, and redirects search results to it.

https://phabricator.wikimedia.org/T206712

### CODE REVIEW

File: templates/translate.html.twig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}
+    <p><a href="{{ path('home') }}">{{ msg('pick-another') }}</a></p>
+    <h1 class="filename">File:{{ filename }}</h1>
```
Comment (aezell) [inline templates/translate.html.twig]: Should we have a space after `File:`?

File: templates/search.html.twig [hunk 2]
```diff
@@ -1,5 +1,14 @@
 {% extends 'base.html.twig' %}
 
+{% block title %}
+    <h1>{{ msg('svgtranslate') }}</h1>
+    <p>{{ msg('info') }}</p>
+    <p>
+        {{ msg('beta') }}
```
Comment (aezell) [inline templates/search.html.twig]: Do we need a space after this message and before the link?

File: templates/translate.html.twig (hunk 1 above)
Comment (samwilson) [reply templates/translate.html.twig]: I'm not sure. It's not got a space on the [prototype](https://prtksxna.github.io/svgtranslate-prototype/solarsystem.html), and I guess is mirroring what things look like on [Commons](https://commons.wikimedia.org/wiki/File:Planets2013.svg). I also wasn't sure if 'File:' should be translated, but I think it makes sense not to.

File: templates/search.html.twig (hunk 2 above)
Comment (samwilson) [reply templates/search.html.twig]: There is one (by virtue of the line after this being indented). We don't have [spaceless](https://twig.symfony.com/doc/2.x/templates.html#whitespace-control) turned on in this bit.

Comment (aezell) [reply templates/search.html.twig]: Got ya. HTML/Twig magic.

File: templates/translate.html.twig (hunk 1 above)
Comment (aezell) [reply templates/translate.html.twig]: Oh, I see now. It's mimicing the URL/naming structure used in wiki projects. It's not really for humans. :)

Comment (aezell) [reply templates/translate.html.twig]: Yep. Agreed. I get it now.

Comment (Niharika29) [reply templates/translate.html.twig]: Agree with @mooeypoo. We don't need the "File:" there. @prtksxna any objections?

Comment (samwilson) [reply templates/translate.html.twig]: Do we want to keep `File:` in the URL structure?

Comment (samwilson) [reply templates/translate.html.twig]: >  we are asking the users to put in a filename from Commons without the File: namespace

Actually, it works if they put in a filename with or without, or a whole URL. So maybe that search message is a bit misleading. We chop off anything up to the end of any `File:` that's present in the string entered, and then do the search. (Oh, hmm... well, in #14 that's what might happen!)

**Comment by samwilson** [comment]
This is ready for re-review.

**Comment by aezell** [comment]
Should it still have the [WIP] tag?

**Comment by samwilson** [comment]
Oops! No. Thanks.

I wish GH had an actual WIP flag.

**Comment by aezell** [comment]
Switched to this branch and reloaded the app. I get these errors in the console and the fonts are incorrect.
`http://localhost:8042/assets/i18n/jquery.uls/en.json - 404 (Not Found)
http://localhost:8042/assets/i18n/app/en.json - 404 (Not Found)`

**Comment by samwilson** [comment]
There seem to be some issues with lighttpd URL rewriting in docker. Am fixing.

Thanks for finding!

**Comment by samwilson** [comment]
The Docker fix is #23; you should be getting the same error with current master, and that patch should fix it.

**Comment by aezell** [comment]
That fix seems to fix it in master. Should we rebase this branch so the fix will work here as well?

**Comment by samwilson** [comment]
Oops, I totally thought I'd done that.

I've fixed the search form infusion issue too now.

**Pull request merged**